File: CTDK_Python/062/_062.py
n = int(input("Nhap n: "))

s = 0;
t = n
while(t != 0):
    dv = t % 10
    if dv % 2 == 0:
        s = s + dv
    t = t // 10

print("Tong cac chu so chan: ", s)

File: CTDK/062/062.cpp
#include <iostream>
using namespace std;

int main()
{
	int n;
	cout << "Nhap n: ";
	cin >> n;

	int s = 0;
	int t = n;
	while (t != 0)
	{
		int dv = t % 10;
		if (dv % 2 == 0)
			s = s + dv;
		t = t / 10;
	}
	cout << "Tong cac chu so chan la: " << s << endl;
	return 0;
}